File: 2586-count-the-number-of-vowel-strings-in-range/2586-count-the-number-of-vowel-strings-in-range.cpp
class Solution {
public:
    int vowelStrings(vector<string>& words, int left, int right) {
        int count=0;
        for(int i=left; i<=right; i++)
        {
            char b=words[i][0];
            char e=words[i].back();
            if((b=='a' || b=='e' || b=='i' || b=='o' || b=='u') &&
               (e=='a' || e=='e' || e=='i' || e=='o' || e=='u'))
            {
                count++;
            }
        }
        return count;
    }
};

File: 2586-count-the-number-of-vowel-strings-in-range/2586-count-the-number-of-vowel-strings-in-range.java
class Solution {
    public int vowelStrings(String[] words, int left, int right) {
        int count=0;
        for(int i=left; i<=right;i++)
        {
            char b=words[i].charAt(0);
            char e=words[i].charAt(words[i].length()-1);
            if((b=='a' ||b=='e' || b=='o' || b=='u' || b=='i') &&
              (e=='a' || e=='e' || e=='o' || e=='u' || e=='i'))
            {
                count++;
            }
        }
        return count;
    }
}